sensu/plugins/check-glance-store.py

```diff
@@ -16,10 +16,12 @@
 from keystoneclient.v2_0 import client as ksclient
 
 def is_remote_image(image):
-    if image.get('location'):
-        http_re = re.compile(r'^(http|https|ftp)://')
-        return http_re.match(image.location)
-    return False
+    if image.get('locations'):
+        for location in image.locations:
```

Comment:
You can simplify this with:
`for location in image.get('locations', []):`

If `locations` key doesn't exist, the get call will return the empty list instead, and the loop will be skipped. No need to have an if statement first.